src/RefractDataStructure.cc

```diff
@@ -41,6 +41,9 @@ namespace drafter {
         if (ta & mson::FixedTypeAttribute) {
             attr->push_back(refract::IElement::Create(SerializeKey::Fixed));
         }
+        if (ta & mson::NullableTypeAttribute) {
+            attr->push_back(refract::IElement::Create(SerializeKey::Nullable));
+        }
 
```

Comment:
@pksunkara this is likely because I was branched off shared/refract then rebased to develop, I'll clean this all up now though.